docs/editor/extension-gallery.md

```diff
@@ -71,13 +72,15 @@ For example, typing 'python' will bring up a list of Python language extensions:
 
 ![python extensions](images/extension-gallery/extensions-python.png)
 
+If you already know the exact identifier for an extension you're looking for, you can use the `@id:` prefix, like `@id:octref.vetur`. Additionally, to filter or sort results, you can use [filter](#extensions-view-filters) and [sort](#sorting) commands, detailed below.
+
 ## Manage extensions
 
 VS Code makes it very easy to manage your extensions. You can install, disable, update, and uninstall extensions through the Extensions view, the **Command Palette** (commands have the **Extensions:** prefix) or command line switches.
 
 ### List installed extensions
 
-By default, the Extensions view will show your installed extensions. You can also use the **Show Installed Extensions** command, available in the **Command Palette** (`kb(workbench.action.showCommands)`) or the **More** (`...`) dropdown, to clear any text in the search box and show the list of installed extensions.
+By default, the Extensions view will show the extensions you currently have enabled. You can also use the **Show Installed Extensions** command, available in the **Command Palette** (`kb(workbench.action.showCommands)`) or the **More** (`...`) dropdown, to clear any text in the search box and show the list of installed extensions, including those that have been disabled.
```

Comment:
lets talk about how we show the split of enabled, recommended and disabled. And then follow it up with something like 'if you want to see a complete list of your installed extensions..`